### ISSUE TITLE
HiDPI support

### ISSUE THREAD

**Issue by skyline75489** [McYoloSwagHam/win3wm#11]
Right now the support for HiDPI and DPI changing event seems missing.

**Comment by McYoloSwagHam** [comment]
Could you explain what you mean?
Do you mean the situation when you move a window from one monitor to another and they have different DPIs?

**Comment by skyline75489** [comment]
Sorry I wasn't being clear. I mean I was changing the DPI of the current monitor. Like from 100% to 200%.

**Comment by McYoloSwagHam** [comment]
Is this something you do commonly, changing DPIs of your monitor?
I can't really see a common use case for changing DPIs of your monitor as you work. 
Regardless, when crash handling and restarting with memory/layout will be implemented, I'm sure this problem will disappear since WinWM would just restart with the new DPI

**Comment by skyline75489** [comment]
Yeah.. my use case might be unusual. The DPI change happens when I remote login a Windows machine using a device that has different DPI than that machine via RDP.

**Comment by McYoloSwagHam** [comment]
Update: Upon further inquiry, this is harder than it looks, WM_DPICHANGED does not supply the HMONITOR whose DPI changed making it very hard to update the windows who are affected by the DPI change.

And using verify_workspace seems to fix the workspace (mod+r)
